GitHub.Collectors/Config/GitHubConfigManager.cs

```diff
@@ -79,6 +81,20 @@ public string GetApiDomain()
             }
             return apiDomain;
         }
+
+        public string GetCollectorIdentity()
+        {
+ 
+            string collectorIdentity = string.Empty;
+            try
+            {
+                collectorIdentity = this.collectorIdentityToken.Value<string>();
```

Comment:
Since this can be null, I think you want to check against null first. I know that exception "eating" handles that scenario as well, but that way it would be more elegant?